Teaching/Miscellaneous/MATMUL.md

```diff
@@ -2,36 +2,9 @@
 ## Background
 Matrix multiplication is a fundamental operation in numerous computational and mathematical tasks. This operation is fundamental in many areas, including graphics rendering, scientific simulations, machine learning, and more. Given the significance of matrix multiplication these various fields, failing to implement it with performance in mind can pose a severe computational bottleneck.
 
-A naïve implementation of this algorithm (shown below) exhibits a time complexity of O(n^3) for two n × n matrices. 
-``` python
-import sys, random
-from tqdm import tqdm
-from time import *
+A naïve implementation of this algorithm [linked here](https://gist.github.com/tejasexpress/cbb793628b05c28a6f531b6ebcb8713b) exhibits a time complexity of O(n^3) for two n × n matrices. 
```

Comment:
What I meant by linking was linking to a Wikipedia page for example, not to a complete code snipped. Just describe the algorithm, the interviewee should figure out how to implement it and can choose a language. Also, you can link to a resource that explains time complexity and how to calculate it.